zio-http/src/main/scala/zhttp/service/Server.scala

```diff
@@ -159,6 +171,7 @@ object Server {
   def error[R](errorHandler: Throwable => ZIO[R, Nothing, Unit]): Server[R, Nothing] = Server.Error(errorHandler)
   def ssl(sslOptions: ServerSSLOptions): UServer                                     = Server.Ssl(sslOptions)
   def acceptContinue: UServer                                                        = Server.AcceptContinue(true)
+  def allocator(allocator: Option[PooledByteBufAllocator]): UServer                  = Allocator(allocator)
```

Comment:
```suggestion
  def allocator(allocator: PooledByteBufAllocator): UServer                  = Allocator(allocator)
```

If you are passing the allocator, then why would it be inside an Option?